bocoup/aria-at-app
# Executing Test: Tester View: Test Conflict Resolution, Pt 1

- Run `yarn db-migrate:dev`
- Run `yarn`
- Run `yarn dev`
- Note that when running from dev configuration, your issues will be created here: https://github.com/bocoup/aria-at/issues.
- Open the app at http://localhost:3000
- Sign out of the app
- In another tab, go here: https://github.com/settings/applications, and revoke your authentication for "ARIA-AT Report API (Development)"
- Return to the app and click "Sign in with Github"
- Authorize
- Once back at the app, make sure you have some ATs set up. 
- Go to http://localhost:3000/cycles, click "Initiate"
- Create a "Test Cycle Configuration"
- Check the checkboxes and assign yourself to each of the 
Test Plans"
- Click "Initiate Cycle"
- Go to http://localhost:3000/test-queue
- Click "Contribute Tests" 
- Click the name of any "Test Plan". If none are clickable, click "Assign Me" then click the name.
- You're now in the test and can actually start testing this feature 🙄 

1. Observe that there are no Status notifications on the page. 
1. Click "Raise an issue", observe that you are now at the "Create an issue" prompt. 
1. Observe there is a form and two buttons: "Cancel" and "Submit new issue"
1. Click "Cancel". Observe that the prompt closes. 
1. Click "Raise an issue", observe that you are now at the "Create an issue" prompt. 
1. Make any changes you want and click "Submit new issue"
1. Observe that you are now at the "Issue created" view and are presented with a message that says "Issue #XX created" where "XX" is the issue number that was created. Click "Done", observe that the prompt closes. 
1. Click "Raise an issue", observe that you are now at the "Review existing issues
" prompt, and shown a list that has your issue in it. 
1. Click "My issue exists in this list", observe that the prompt closes. 
1. Click "Raise an issue"
1. Click "My issue is not in this list", observe that you are at the "Create an issue" prompt. 
1. Click "Back to issues list", observe that you are at the "Review existing issues
" prompt.
1. Click the "X" in the upper right of the prompt. 
1. Refresh the test page
1. Observe that there is now a "status bar" that says "This test has open issues"
1. Click the "Review issues" button, observe that you are now at the "Review existing issues" prompt.
1. Scroll to the "See on github.com" link and click it.
1. Close the issue on Github
1. Return to the ARIA-AT test page and refresh. 
1. Observe that the status bar is gone. 
1. Click "Raise an issue", observe that you are now at the "Create an issue" prompt.
